inc/invitation.class.php

```diff
@@ -203,10 +203,19 @@ public function prepareInputForUpdate($input) {
       return $input;
    }
 
+   /**
+    * Finds the token
```

Comment:
It does not finds a token. It finds an invitation in the database which has the token given in argument.

FYI: getFromDB*() methods build a SQL query with a WHERE statement. IF a row (and only one row) is found, then it is used to populate the object.